### PR TITLE
Create agg_is_at_least.md

### PR DESCRIPTION
**Pull request by igor-aptos**

### CODE REVIEW

File: aips/agg_is_at_least.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Risks and Drawbacks
+
+## Security Considerations
+
+None, provides the same functionality as before, just more efficiently
+
+## Future Potential
+
+Once traits are deployed, `is_at_most`/`is_equal` utility methods will be provied as well. Until then, they can be derived from this one (assuming +1 doesn't overflow):
```
Comment (movekevin) [inline aips/agg_is_at_least.md:68]: provied -> provided

Comment (igor-aptos) [reply aips/agg_is_at_least.md]: fixed

File: aips/agg_is_at_least.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+aip: X
+title: Introduce aggregator_v2::is_at_least native API
```
Comment (sherry-x) [inline aips/agg_is_at_least.md:3]: "Introduce native API to evaluate aggregator value is greater or equal to given amount" instead?

Comment (igor-aptos) [reply aips/agg_is_at_least.md]: updated

**Review by sherry-x — APPROVED** [review]

**Pull request merged**